### ISSUE TITLE
Lupopover - focus and hover

### ISSUE THREAD

**Issue by AlexiaAlarza** [LuccaSA/lucca-front#1393]
Allow focus + hover on LuPopover component

**Comment by lucienbertin** [comment]
popover triggers should be complitely reworked
